src/actions.ts

```diff
@@ -31,12 +31,41 @@ export function getDependencies(
   return { keys, selected, selections };
 }
 
-export async function installTypes(dependencies: string[], { toDev = false, selections, pwd = '', concurrency = 1 }) {
-  const { stdout: yarnPath } = await shell('which yarn');
+function getYarnVersion() {
+  return shell('yarn --version')
```

Comment:
Use async/await

Reply:
I actually got a exception in the typescript compiler trying to use async/await for that, and gave up really quick.  I can try again, I guess.